### DISCUSSION TITLE
Option for Encrypting Archive without Compression

### DISCUSSION THREAD

**Discussion by CelebStalker** [miurahr/py7zr#562] [Q&A]
I have pre compressed photos and videos I'd like to archive, but wouldn't like the program to spend time on compression. Is there an option to just apply encryption without file compression in the program?

**Comment by miurahr** [comment]
py7zr accept a filter value. 
https://py7zr.readthedocs.io/en/latest/api.html#possible-filters-value

You can find several examples like

> 7zAES + LZMA
> [{'id': FILTER_LZMA}, {'id': FILTER_CRYPTO_AES256_SHA256}]
> 
> 7zAES + Deflate
> [{'id': FILTER_DEFLATE}, {'id': FILTER_CRYPTO_AES256_SHA256}]
> 
> 7zAES + BZip2
> [{'id': FILTER_BZIP2}, {'id': FILTER_CRYPTO_AES256_SHA256}]
> 
> 7zAES + ZStandard
> [{'id': FILTER_ZSTD}, {'id': FILTER_CRYPTO_AES256_SHA256}]

What is  a 7zAES only array?

**Comment by CelebStalker** [comment]
I'm unable to find filter that allows encryption with zero compression (like a direct store). 
Any suggestion as to which filter allows to do what the screenshot allows?

![Screenshot_20221113_010022](https://user-images.githubusercontent.com/113247477/201510976-5a52d332-939d-4fc9-a019-53e2d62b0f33.png)

**Comment by miurahr** [comment]
``FILTER_COPY``

**Comment by CelebStalker** [comment]
Thank You
